autogpt_platform/frontend/src/app/(platform)/admin/execution-analytics/components/AnalyticsResultsTable.tsx

```diff
@@ -0,0 +1,292 @@
+"use client";
+
+import React, { useState } from "react";
+import { Button } from "@/components/__legacy__/ui/button";
+import { Badge } from "@/components/__legacy__/ui/badge";
+import { DownloadIcon, EyeIcon } from "@phosphor-icons/react";
+import { Copy } from "lucide-react";
+import { useToast } from "@/components/molecules/Toast/use-toast";
```

Comment:
Let's use [`<Button />`](https://design.agpt.co/?path=/story/atoms-button--all-variants) and [`<Text />`](https://design.agpt.co/?path=/story/atoms-text--all-variants). These are internal pages, so I'm not too hard pressed, given users won't see this, but I prefer we stop adding dependencies to the old components 🙇🏽